### PR TITLE
Support WSEGVALV

### PR DESCRIPTION
**Pull request by GitPaean**

### CODE REVIEW

**Comment by GitPaean** [comment]
jenkins build this opm-common=1257 please

**Comment by GitPaean** [comment]
jenkins build this opm-common=1257 please

File: compareECLFiles.cmake [hunk 1]
```diff
@@ -378,6 +378,13 @@ add_test_compareECLFiles(CASENAME editnnc_model2
 			  REL_TOL ${rel_tol}
 			  DIR model2)
 
+add_test_compareECLFiles(CASENAME wsegsicd
+			  FILENAME TEST_WSEGSICD.DATA
+			  SIMULATOR flow
+			  ABS_TOL ${abs_tol}
+			  REL_TOL ${rel_tol}
+			  DIR wsegsicd)
```
Comment (akva2) [inline compareECLFiles.cmake]: not necessary to specify DIR if it matches the casename.

Comment (GitPaean) [reply compareECLFiles.cmake]: okay. I pushed it to the wrong branch even. Thanks for commenting.

**Comment by GitPaean** [comment]
jenkins build this opm-common=1257 please

**Comment by GitPaean** [comment]
jenkins build this opm-common=1257 please

**Comment by GitPaean** [comment]
jenkins build this opm-common=1257 please

**Comment by GitPaean** [comment]
jenkins build this opm-common=1257 please

**Comment by GitPaean** [comment]
jenkins build this opm-common=1257 please

**Comment by GitPaean** [comment]
jenkins build this opm-common=1257 please

**Comment by atgeirr** [comment]
The code looks good, had to double-check the pressure drop formulation though. A comment to point out that velocity = mass rate / (density * area) would help.

**Comment by GitPaean** [comment]
jenkins build this opm-common=1257 please

**Comment by GitPaean** [comment]
jenkins build this opm-common=1257 please

**Pull request merged**